tests/test_jirakit.py

```diff
@@ -16,5 +16,16 @@ def testCycle(self):
         c = jirakit.cycles()
         self.assertEqual(c[0], 'S14')
 
+    def testIssueUrl(self):
+        try:
+            # Python 3
+            from urllib.parse import urljoin
+        except ImportError:
+            # Python 2
+            from urlparse import urljoin
+        key = "SIM-1289"
+        comp_url = urljoin(jirakit.SERVER, "browse/" + key)
```

Comment:
This test is a bit tricky because it duplicates the code in the thing that is being tested.